docs/docs/segment-exit.md

```diff
@@ -50,3 +50,14 @@ Displays the last exit code or that the last command failed based on the configu
 - `.Meaning`: `string` - the textual meaning linked to exit code (if applicable, otherwise identical to `.Code`)
 
 [templates]: /docs/config-templates
+
+** Nu >= 0.60.0 **
```

Comment:
Adjusting the [prompt installation guide](https://github.com/JanDeDobbeleer/oh-my-posh/blob/main/docs/docs/install-prompt.mdx) will be easier. You can add the flag to the command listed there.